### PR TITLE
Feature/update from may

### PR DESCRIPTION
**Pull request by gpsgp**
Generate new docs and add how-to guide.
Does not currently validate using swagger editor - needs to be fixed where API docs is generated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # apidocs
 Apidocs is a project containing Open API documentation for GpsGate public APIs (https://gpsgate.com/developer)
 
-# Validation
-Always valid the REST API using the Swagger Editor (http://editor.swagger.io/) before committing/merging
+# Generate REST API
```
Comment (bjornhandersson) [inline README.md:4]: 👍

**Review by bjornhandersson — APPROVED** [review]
What has to be done to get the docs pass the validation? 
Update `Nancy.Swagger`? 
Unfortunately Nancy is no longer maintained.

**Comment by gpsgp** [comment]
> What has to be done to get the docs pass the validation? Update `Nancy.Swagger`? Unfortunately Nancy is no longer maintained.

image
We need to:
1. Change path for one of the following (Equivalent paths are not allowed):
/applications/{applicationid:int}/users/{userid:long}
/applications/{applicationid:int}/users/{username:string}
2. Fix unique operationIds (4 occurences)
3. Lowercase Description and Url in externalDocs

**Comment by bjornhandersson** [comment]
> > What has to be done to get the docs pass the validation? Update `Nancy.Swagger`? Unfortunately Nancy is no longer maintained.
> 
> image We need to:
> 
> 1. Change path for one of the following (Equivalent paths are not allowed):
>    /applications/{applicationid:int}/users/{userid:long}
>    /applications/{applicationid:int}/users/{username:string}
> 2. Fix unique operationIds (4 occurences)
> 3. Lowercase Description and Url in externalDocs

OK, this is nothing new and it "works" so I guess we have to accept those warnings since changing them would break some APIs.

**Pull request merged**